gap_analysis/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(2015)>
+# <AUTHOR(Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng)>
```

Comment:
The right format is like this?
<Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng(S)>